README.md

```diff
@@ -26,11 +26,12 @@ fn main() {
     let mut filter_output = vec![0.0f32; input_img.len()];
 
     let device = oidn::Device::new();
-    let mut filter = oidn::RayTracing::new(&device);
-    // Optionally add float3 normal and albedo buffers as well
-    filter.set_srgb(true)
-        .set_img_dims(input.width() as usize, input.height() as usize);
-    filter.execute(&input_img[..], &mut filter_output[..]).expect("Filter config error!");
+    oidn::RayTracing::new(&device)
+        // Optionally add float3 normal and albedo buffers as well
+        .srgb(true)
+        .set_image_dimensions(input.width() as usize, input.height() as usize);
```

Comment:
Should be `image_dimensions`? based on the change to the filter API

Reply:
Yes, I fixed the code in the README.